### PR TITLE
Consul Tags on Services

### PR DESCRIPTION
**Pull request by codyja**
Hi,
Great project and very useful for load testing consul clusters! Thank you!

In this PR, I tried adding a new optional flag called `-tags` that will take a comma separated list of flags to add onto each service. 

We have some load balancers (Fabio and Traefik) that monitor services in consul looking for specific tags on services. Then these matched services get added as routes in the load balancers. To help us do a more real world load tests, we thought it would be great to more closely mimic our production consul clusters with the load balancers in the mix also watching services.

### CODE REVIEW

**Comment by pierresouchay** [comment]
@codyja Still small error in Travis, ping us if you need help

File: service.go [hunk 1]
```diff
@@ -32,14 +33,22 @@ func DeregisterServices(client *consul.Client, serviceName string) error {
...
 	log.Printf("Registering %d %s instances...\n", count, serviceName)
 
 	checksTTL := flapInterval * 3
 	if checksTTL == 0 {
 		checksTTL = 10 * time.Minute
 	}
 
+	var tags []string
+	if serviceTags != "" {
+		tagsSplit := strings.Split(serviceTags, ",")
```
Comment (ShimmerGlass) [inline service.go]: You can remove the `for` and just do `tags = strings.Split(serviceTags, ",")`

**Review by ShimmerGlass — CHANGES_REQUESTED** [review]

**Comment by ShimmerGlass** [comment]
Could you rebase your PR to fix the build issue ?

File: service.go (hunk 1 above)
Comment (codyja) [reply service.go]: Ah, that is much better and I understand why now too. Thanks

**Comment by codyja** [comment]
I believe I rebased pulling in the latest changes, but I'm struggling to get past the build failure in travis for some reason still.

**Comment by codyja** [comment]
@pierresouchay @ShimmerGlass , it took me a little while but it looks like that last build passed about a year ago on go 1.13. This latest build was trying to updated dependencies with that `-u` flag on `go get` and it was pulling in a consul package with some issues. Let me know if you think removing that `-u` flag was acceptable or not.

Also, should I do a git tag here also?

**Comment by ShimmerGlass** [comment]
Thx for taking the time to debug this. Indeed, we shouldn't have a `go get -u` in the build I'm not sure how the go mod PR passed.
In fact, now that we are using modules `go get` is unnecessary.
I'll merge your PR and fix that on my end, as well as creating a new release.

**Review by ShimmerGlass — APPROVED** [review]

**Comment by codyja** [comment]
Awesome, thank you much!

**Pull request merged**